RecoHI/HiJetAlgos/plugins/HiFJGridEmptyAreaCalculator.cc

```diff
@@ -171,11 +171,11 @@ void HiFJGridEmptyAreaCalculator::produce(edm::Event& iEvent, const edm::EventSe
 void HiFJGridEmptyAreaCalculator::calculateGridRho(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
   vector<vector<double>> scalarPt(ny_, vector<double>(nphi_, 0.0));
 
-  edm::Handle<reco::PFCandidateCollection> pfCands;
+  edm::Handle<reco::CandidateView> pfCands;
   iEvent.getByToken(pfCandsToken_, pfCands);
-  const reco::PFCandidateCollection* pfCandidateColl = pfCands.product();
+  const auto& pfCandidateColl = pfCands.product();
   for (unsigned icand = 0; icand < pfCandidateColl->size(); icand++) {
```

Comment:
Maybe we should take this opportunity to move to a range based for loop.  I think that will execute more efficiently than accessing via an `at()`

Reply:
done in b3287056ba0fb3fd5aefd75cb0d60156237e35ca